database/src/main/resources/db/migration/V1.77.0__RHCLOUD-25998_add_drawer_notification_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public.drawer_notification (
+    user_id varchar(50) NOT NULL,
+    org_id text NOT NULL,
+    event_id uuid NOT NULL,
+    created timestamp NOT NULL,
+    read boolean NOT NULL default false,
```

Comment:
Sorry, I forgot something in the Jira ticket 😅 Since this field will be updated from the REST API, I think the `drawer_notification` table needs an `id uuid` column which would be the PK.

Reply:
Ok